lib/bitmap/bitmap.nit

```diff
@@ -130,9 +130,6 @@ class Bitmap
 		# =============== Bitmap header ================
 		for x in [0..13] do
 			var b = fileReader.read_byte
-			if b == null then
-				return
-			end
 			bitmap_header[x] = b.to_i
```

Comment:
Since read_byte now returns an `Int`, the `to_i` here is not necessary

Reply:
It can however return -1 in case of error, this should be checked however

Reply:
I only removed the warning, I will let you make the fixes.

Reply:
Fair enough